JournalMonitor/JournalMonitor.cs

```diff
@@ -1110,11 +1110,29 @@ public static List<Event> ParseJournalEntry(string line)
                                 string from = getString(data, "From");
                                 string channel = getString(data, "Channel");
                                 string message = getString(data, "Message");
+                                string source = null;
```

Comment:
You don't need to init it to null, in C# everything is iniitialized with its default initializer, which is "" for strings